src/LINEBot.php

```diff
@@ -1251,4 +1267,52 @@ public function testWebhookEndpoint($endpoint)
             'endpoint' => $endpoint,
         ]);
     }
+
+    /**
+     * Get the per-unit statistics of how users interact
+     * with push messages and multicast messages
+     * sent from your LINE Official Account.
+     *
+     * @param string $customAggregationUnit Name of aggregation unit
+     * @param string $from Start date of aggregation period
+     * @param string $to End date of aggregation period
+     * @return Response
+     */
+    public function getUserInteractionStatisticsPerUnit($customAggregationUnit, $from, $to)
+    {
+        return $this->httpClient->get($this->endpointBase . '/v2/bot/insight/message/event/aggregation', [
+            'customAggregationUnit' => $customAggregationUnit,
+            'from' => $from,
+            'to' => $to,
+        ]);
+    }
+
+    /**
+     * Get the number of aggregation units used this month.
+     *
+     * @return Response
+     */
+    public function getNumberOfUnitsUsedThisMonth()
+    {
+        return $this->httpClient->get($this->endpointBase . '/v2/bot/message/aggregation/info');
+    }
+
+    /**
+     * Get the name list of units used this month for statistics aggregation.
+     *
+     * @param string|null $limit
+     * @param string|null $start
+     * @return Response
+     */
+    public function getNameListOfUnitsUsedThisMonth($limit = null, $start = null)
+    {
+        $data = [];
+        if (isset($limit)) {
+            $data['limit'] = $limit;
+        }
+        if (isset($start)) {
+            $data['start'] = $start;
+        }
```

Comment:
Please rewrite continuation token implementation.
Using this token, you can get all remaining data from server.
See: https://github.com/line/line-bot-sdk-php/blob/210d87af3f5fa3ed7837c3c875e39e1e3caeb9b7/src/LINEBot.php#L109-L140